Rubberduck.CodeAnalysis/Inspections/Concrete/UndeclaredRedimVariableInspection.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Resources.Inspections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    public sealed class UndeclaredRedimVariableInspection : InspectionBase
+    {
+        public UndeclaredRedimVariableInspection(RubberduckParserState state) 
+            : base(state) { }
+
+        protected override IEnumerable<IInspectionResult> DoGetInspectionResults()
+        {
+            return State.DeclarationFinder.UserDeclarations(DeclarationType.Variable)
+                .Where(item => item.IsRedimVariable && !IsIgnoringInspectionResultFor(item, AnnotationName))
+                .Select(item => new DeclarationInspectionResult(this, string.Format(InspectionResults.UndeclaredRedimVariableInspection, item.IdentifierName), item));
```

Comment:
It looks like we do have `UndeclaredRedimVariableInspection` defined in the resx file but in the `.designer`, it's apparently under `InspectionName`? See `InspectionNames.Designer.cs`, line 633. That might be contributing to the build failure.